src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/ValidationResultExceptionConverter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class ValidationResultExceptionConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return typeof(Exception).IsAssignableFrom(objectType);
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            try
+            {
+                return serializer.Deserialize(reader, objectType);
+            }
+            catch
+            {
+                return new Exception("This exception could not be deserialized. Please try running validations again.");
```

Comment:
Never throw a general exception. You should throw Deserialization exception, and add the original exception caught as an inner exception so it can be logged.

Reply:
Maybe add few comments ... In what scenarios the ExceptionConverter will be used?  How the returned exception will be used ?  Why not let it throw?